classes/assets/StylesheetManager.php

```diff
@@ -53,11 +53,18 @@ public function register(
         $media = self::DEFAULT_MEDIA,
         $priority = self::DEFAULT_PRIORITY,
         $inline = false,
-        $server = 'local'
+        $server = 'local',
+        $needRtl = true
     ) {
+        $fullPath = $this->getFullPath($relativePath);
+        $rtlFullPath = $this->getFullPath(str_replace('.css', '_rtl.css', $relativePath));
+        $context = Context::getContext();
+        $isRTLed = $context->language->is_rtl;
         if ('remote' === $server) {
             $this->add($id, $relativePath, $media, $priority, $inline, $server);
-        } elseif ($fullPath = $this->getFullPath($relativePath)) {
+        } else if ($needRtl && $isRTLed && $rtlFullPath) {
```

Comment:
@maximebiloe I've checked AssetsCoreTest and I can't find any problem in it. needRTL is true by default but another condition is needed to make it works.